async/src/main/java/org/odk/collect/async/CoroutineAndWorkManagerScheduler.kt

```diff
@@ -57,14 +59,23 @@ class CoroutineAndWorkManagerScheduler(
         inputData: Map<String, String>,
         networkConstraint: Scheduler.NetworkType?
     ) {
-        val constraintNetworkType = when (networkConstraint) {
+        val networkRequest = NetworkRequest.Builder().apply {
```

Comment:
I think we still need to add the `NetworkCapabilities.NET_CAPABILITY_INTERNET` capability here right?